_posts/wiki/2020-03-23-robotic-skin.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Robotic Skin
```

Comment:
better title needed

Reply:
Maybe the poster title: **Self-Contained Kinematic Calibration of Flexible Whole-Body Artificial Skin for Collaborative Robotics**? @watakandai @peasant98 any other ideas for a compelling title?